api.yaml

```
@@ -162,6 +162,9 @@ paths:
         Fetching a transaction by hash is considered an Explorer Method
         (which is classified under the Future Work section).
 
+        This method can be used to let consumers to paginate results when the 
+        block trasactions count is too big to be returned in a single [BlockResponse](#/components/schemas/BlockResponse). 
```

```suggestion
        block trasactions count is too big to be returned in a single BlockResponse. 
```

Links in the descriptions break the formatting 😢  Hopefully, we can fix this issue and do one large pass on adding links in the future.